packages/@aws-cdk/aws-synthetics/lib/schedule.ts

```diff
@@ -13,8 +13,11 @@ export class Schedule {
   }
 
   /**
-   * Construct a schedule from a literal schedule expression. The expression must be in a `rate(number units)` format.
-   * For example, `Schedule.expression('rate(10 minutes)')`
+   * Construct a schedule from a literal schedule expression. The expression must be in a `rate(number units)` or `cron(0 12 * * * *)` format.
+   * With a `rate()` expression only intervals of < 60 minutes can be used.
+   * For example:
+   * `Schedule.expression('rate(10 minutes)')`
+   * `Schedule.expression('cron(0 6 ? * MON-FRI *)')`
    *
```

Comment:
As @NetaNir mentioned, I'd like to see a simple validation on this statement for the rate expression. Just a string parsing for "rate" at the beginning of the expression and ensuring the minute equivalent is at most 60. Do you think you can add this change here?

Reply:
Not addressed